<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppCaffe</class>
 <widget class="QMainWindow" name="AppCaffe">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>592</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>789</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AppCAffe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>asset/gambar.png</normaloff>asset/gambar.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QmainWindow#AppCaffe {
    background-image: url(C:/Users/user/Documents/tugas akhir semester 2/PBO/asset/bg.jpeg);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">#centralwidget {
    background-image: url(C:/Users/user/Documents/tugas akhir semester 2/PBO/asset/bg.jpeg);
}
</string>
   </property>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Copyright Â© 2023 InpoNgopi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 italic 28pt &quot;Arial&quot;;</string>
    </property>
    <property name="text">
     <string>DAFTAR MENU</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>771</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout" columnstretch="3,5,1">
       <item row="2" column="0">
        <widget class="QLabel" name="label_14">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Mocacino</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_1">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 12.000</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_15">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Americano</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_16">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Latte</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 15.000</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_4">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 13.000</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_17">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Espresso</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="spinBox_5">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_18">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Mocha</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_5">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 11.000</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 10.000</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_13">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Cappucino</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_1">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 10.000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="spinBox_3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="spinBox_4">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="spinBox_6">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="3,0,0">
       <item row="0" column="1">
        <widget class="QLabel" name="label_7">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 20.000</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_19">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Pasta</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_20">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Kebab</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QSpinBox" name="spinBox_7">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_8">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 30.000</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBox_8">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_21">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Pizza</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_22">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Hotdog</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_9">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 25.000</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="spinBox_9">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="spinBox_10">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_23">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Dinsum</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_11">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 17.000</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_24">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Steak</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QSpinBox" name="spinBox_11">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="spinBox_12">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_12">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 20.000</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font: 11pt &quot;Arial&quot;;
color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>                         Rp 30.000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Arial Narrow&quot;;
border: none;
QTableView::item{padding:0px;}
background-color: rgb(170, 85, 0);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>58</width>
      <height>19</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Pesanan</string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>231</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_25">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nama Pemesan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pemesan">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_26">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Nomor Pemesan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="no_pesan">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_27">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Jumlah Uang</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="jumlah_uang">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="proses">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 85, 0);
color: rgb(255, 255, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Proses</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>70</y>
      <width>81</width>
      <height>53</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;
text-decoration: underline;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Makanan</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>91</width>
      <height>53</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Arial&quot;;
text-decoration: underline;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Minuman</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label_54</zorder>
   <zorder>label_30</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_28</zorder>
   <zorder>listView</zorder>
   <zorder>tableWidget</zorder>
   <zorder>label_33</zorder>
   <zorder>label_32</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
